libpod/runtime_pod_namespaces_linux.go

```diff
@@ -0,0 +1,88 @@
+// +build linux
+
+package libpod
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/docker/docker/pkg/mount"
+	"github.com/docker/docker/pkg/symlink"
```

Comment:
Might clash with https://github.com/containers/libpod/pull/5209

Reply:
Yes. We want to get rid of direct dependencies on docker/docker.